emva1288/camera/routines.py

```diff
@@ -130,6 +130,61 @@ def get_radiance(exposure, wavelength, photons, pixel_area, f_number):
     return r
 
 
+def get_tile(arr, height, width):
+    """From an array with a pattern, save execution time in np.tile function by
```

Comment:
implementations details don't go into the docstring, they go in comments if needed